kymatio/scattering1d/filter_bank.py

```diff
@@ -409,14 +409,13 @@ def compute_xi_max(Q):
     return xi_max
 
 
-def compute_params_filterbank(sigma_low, Q, r_psi=math.sqrt(0.5), alpha=5.):
+def compute_params_filterbank(sigma_min, Q, r_psi=math.sqrt(0.5), alpha=5.):
     """
     Computes the parameters of a Morlet wavelet filterbank.
 
     This family is defined by constant ratios between the frequencies and
     width of adjacent filters, up to a minimum frequency where the frequencies
-    are translated.
-    This ensures that the low-pass filter has the largest temporal support
+    are translated. sigma_min specifies the smallest frequential width
     among all filters, while preserving the coverage of the whole frequency
```

Comment:
not sure if the `while preserving the coverage[...]` still makes sense here

Reply:
Yes: if an implementation is CQT-only, limiting `sigma_min` limits `xi_min` and prevents tiling whole frequency axis.

Reply:
Ok so let's skip the last part then, and have `The sigma_min parameter specifies the smallest frequential width among all filters.`

Reply:
If the sentence makes sense it can stay. It's just that I don't fully follow it.
The fourier-signal reciprocity is also not helping with interpreting the words min and max

Reply:
@janden @eickenberg This commit doesn't change the relevance of that line.

In CQT, if we limit `sigma_min`, we also limit the lowest center frequency, so we will not tile the entire frequency axis. If docs make no mention, it can mislead that "lower `J`" -> "less coverage".

That's why the line was there in the first place, was it not?

Reply:
This sentence doesn't make sense logically. Before it was “This ensures A, while satisfying B.” Now it's “A specifies X, while satisfying B.” The first is a constrained guarantee, while the second is… I don't know?

Reply:
Agreed it's a bit awkward, but I think the idea's conveyed. The alternative is `sigma_min limits the smallest frequential width`, which is actually more accurate, `min(sigma1s) == sigma_min` isn't guaranteed.

Also I didn't get pinged for this, Github's notifs aren't reliable - @janden to be sure.

Reply:
Clarifying, `sigma_min` is guaranteed with `Q > 1` via "intermediate" wavelets.